emily/api-definition/rust-config.json

```diff
@@ -3,5 +3,6 @@
   "packageVersion": "0.1.0",
   "library": "reqwest",
   "supportAsync": "true",
-  "withAWSV4Signature": "true"
+  "withAWSV4Signature": "true",
+  "useSingleRequestParameter": "true"
```

Comment:
This makes the request packaged as a single struct as opposed to a flat function call to make the request. This is what lets us deserialize so easily.

Reply:
I'm not sure I follow, what is happening now and what was happening before? Just curious.